src/pages/MainPage/MainPage.jsx

```diff
@@ -1,21 +1,25 @@
-import NavBar from '../../components/common/NavBar/NavBar';
+import Page from '../../components/common/Page/Page';
 import HeroSection from './HeroSection';
 import AboutSection from './AboutSection';
 import CitiesSection from './CitiesSection';
-import Footer from '../../components/common/Footer/Footer';
 import BecomeGuide from './BecomeGuide';
 import OurGuidesSection from './OurGuidesSection';
 import ContactsSection from './ContactsSection';
 import TravelTips from './TravelTips';
 import MenuBottom from './MenuBottom';
+import { Helmet } from 'react-helmet';
 
 const MainPage = () => {
   return (
     <>
-      <header>
-        <NavBar />
-      </header>
-      <main>
+      <Helmet>
```

Comment:
Each page should have its own description and unique title, so let's remove this code from each page implementation and put it to `Page` component